helm/platform-health/templates/deployment.yaml

```
@@ -0,0 +1,34 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}--{{ .Chart.Name }}
+  {{- $deployment := .Values.deployment | default dict }}
+  {{- $labels := mergeOverwrite (.Values.common_labels | default dict) ($deployment.labels | default dict) | toYaml | nindent 4 }}
+  labels: {{ $labels }}
+  {{- $annotations := mergeOverwrite (.Values.common_annotations | default dict) ($deployment.annotations | default dict) | toYaml | nindent 4 }}
+  annotations: {{ $annotations }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}--{{ .Chart.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}--{{ .Chart.Name }}
+    spec:
+      volumes:
+        - name: config
+          configMap:
+            name: platform-health
+      containers:
```

```suggestion
            name: platform-health
      securityContext:
        fsGroup: 1000
        runAsGroup: 1000
        runAsNonRoot: true
        runAsUser: 1000
        seccompProfile:
          type: RuntimeDefault
      containers:
```
I'm unsure if it's best practice to parametrise the uids/gids, or perhaps just the entire `securityContext` block?

---

I will be hardcoding these as it seems the most secure option moving forward.